[2023-05-10 03:51:55,811] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: trigger_dags_in_order.trigger_data_processing_dag manual__2023-05-10T03:51:53.327966+00:00 [queued]>
[2023-05-10 03:51:55,846] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: trigger_dags_in_order.trigger_data_processing_dag manual__2023-05-10T03:51:53.327966+00:00 [queued]>
[2023-05-10 03:51:55,847] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-05-10 03:51:55,847] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2023-05-10 03:51:55,847] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-05-10 03:51:55,877] {taskinstance.py:1259} INFO - Executing <Task(TriggerDagRunOperator): trigger_data_processing_dag> on 2023-05-10 03:51:53.327966+00:00
[2023-05-10 03:51:55,881] {standard_task_runner.py:52} INFO - Started process 636 to run task
[2023-05-10 03:51:55,886] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'trigger_dags_in_order', 'trigger_data_processing_dag', 'manual__2023-05-10T03:51:53.327966+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpqj_7qmu2', '--error-file', '/tmp/tmprl9p5dpg']
[2023-05-10 03:51:55,891] {standard_task_runner.py:77} INFO - Job 113: Subtask trigger_data_processing_dag
[2023-05-10 03:51:56,025] {logging_mixin.py:109} INFO - Running <TaskInstance: trigger_dags_in_order.trigger_data_processing_dag manual__2023-05-10T03:51:53.327966+00:00 [running]> on host cd79de2e36bd
[2023-05-10 03:51:56,263] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=trigger_dags_in_order
AIRFLOW_CTX_TASK_ID=trigger_data_processing_dag
AIRFLOW_CTX_EXECUTION_DATE=2023-05-10T03:51:53.327966+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-10T03:51:53.327966+00:00
[2023-05-10 03:51:56,386] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/api/common/experimental/trigger_dag.py:91 DeprecationWarning: Calling `DAG.create_dagrun()` without an explicit data interval is deprecated
[2023-05-10 03:51:56,519] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=trigger_dags_in_order, task_id=trigger_data_processing_dag, execution_date=20230510T035153, start_date=20230510T035155, end_date=20230510T035156
[2023-05-10 03:51:56,607] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-05-10 03:51:56,808] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
